src/components/CardGrid/CardGrid.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import { Card } from '../../components/Card';
+
+const CardWrapperComponent = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 40px;
```

Comment:
from task description  - _padding - as on the Vazco website components_
i think that gap should remain the same too _for inside elements and **outside tiles**_
gap from Vazco website component - `gap: 20px;`

Reply:
We discussed with @caluskrzysztof that the gap should be made a bit bigger